writers_rollingfilewriter.go

```diff
@@ -698,28 +710,12 @@ func (rwt *rollingFileWriterTime) sortFileRollNamesAsc(fs []string) ([]string, e
 	return ss.data, nil
 }
 
-func (rwt *rollingFileWriterTime) getNewHistoryRollFileName(lastRollName string) string {
-	return ""
+func (rwt *rollingFileWriterTime) getNewHistoryRollFileName(lastRoll rollInfo) string {
+	return lastRoll.Time.Format(rwt.timePattern)
 }
 
 func (rwt *rollingFileWriterTime) getCurrentModifiedFileName(originalFileName string, first bool) (string, error) {
-	if first {
-		history, err := rwt.getSortedLogHistory()
-		if err != nil {
-			return "", err
-		}
-		if len(history) > 0 {
-			return history[len(history)-1], nil
-		}
-	}
-
-	switch rwt.nameMode {
-	case rollingNameModePostfix:
-		return originalFileName + rollingLogHistoryDelimiter + time.Now().Format(rwt.timePattern), nil
-	case rollingNameModePrefix:
-		return time.Now().Format(rwt.timePattern) + rollingLogHistoryDelimiter + originalFileName, nil
-	}
-	return "", fmt.Errorf("Unknown rolling writer mode. Either postfix or prefix must be used")
+	return originalFileName, nil
```

Comment:
Yes. The reason this function even existed on the interface was the way roll by time behaved. This is what I described here: https://github.com/cihub/seelog/pull/117#issuecomment-220487537